connector-runtime/connector-runtime-application/src/main/resources/logback-spring.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+  <if condition='property("CONNECTORS_LOG_APPENDER").equalsIgnoreCase("stackdriver")'>
```

Comment:
What about using https://docs.spring.io/spring-boot/docs/current/reference/html/features.html#features.logging.logback-extensions.environment-properties here? This allows an `application.properties` based config.